bowtie/_report.py

```diff
@@ -327,17 +332,27 @@ def generate_badges(self, target_dir: Path, dialect: str):
                 - counts.skipped_tests
             )
             pct = (passed / total) * 100
-            impl_dir = target_dir / f"{lang}-{name}"
-            impl_dir.mkdir(parents=True, exist_ok=True)
             r, g, b = 100 - int(pct), int(pct), 0
-            badge = {
+            badge_per_draft = {
                 "schemaVersion": 1,
                 "label": label,
                 "message": "%d%% Passing" % int(pct),
                 "color": f"{r:02x}{g:02x}{b:02x}",
             }
-            badge_path = impl_dir / f"{label.replace(' ', '_')}.json"
-            badge_path.write_text(json.dumps(badge))
+            comp_dir = target_dir / f"{lang}-{name}" / "compliance"
+            comp_dir.mkdir(parents=True, exist_ok=True)
+            badge_path_per_draft = comp_dir / f"{label.replace(' ', '_')}.json"
+            badge_path_per_draft.write_text(json.dumps(badge_per_draft))
+            badge_supp_draft = {
+                "schemaVersion": 1,
+                "label": "JSON Schema Versions",
+                "message": supported_drafts,
+                "color": "lightgreen",
+            }
+            supp_dir = target_dir / f"{lang}-{name}"
```

Comment:
This is `impl_dir` from before the changes (so you could have left it as-is and just made `comp_dir` a child of `impl_dir`), but if you're going to do it this way now, there should be no need for the `.mkdir` on the next line -- the directory must always exist, it's a parent of hte directory you created on line 342, right?